### PR TITLE
feat:nav documentation

### PR DESCRIPTION
**Pull request by stutisk**
https://user-images.githubusercontent.com/48529714/153300435-430e3b07-f763-42c0-8135-550802b8123d.mp4

### CODE REVIEW

**Review by aryan-13 — COMMENTED** [review]
Looks good to merge!

File: components/navigation/navigation.html [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+                <span class="bar"></span>
+                </a>
+
+            </div>
+            <div class="right-side">
+                <ul>
+                    <li><a href="#"><i class="fa fa-github"></i></a></li>
+                    <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+                    <li><a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
+                    <li><a onclick="myFunction()" href="#"><i class="fa fa-moon-o" aria-hidden="true"></i></a></li>
```
Comment (JantuDeb) [inline components/navigation/navigation.html:36]: Avoid using inline js in html, use `addEventListener()`

**Review by JantuDeb — APPROVED** [review]
Suggested a changes

File: components/navigation/navigation.html (hunk 1 above)
Comment (stutisk) [reply components/navigation/navigation.html:36]: yah had to remove this !thnku

**Pull request merged**